.github/workflows/ci.yml

```diff
@@ -94,6 +99,11 @@ jobs:
         if: matrix.execute-flex-with-symfony-version
         run: composer config extra.symfony.require ${{ matrix.execute-flex-with-symfony-version }}
 
+      - name: Lower minimum stability to test with Symfony 6
+        if: matrix.execute-flex-with-symfony-version == '^6'
+        run: |
+          composer config minimum-stability beta
+
```

Comment:
```suggestion
```

Can be removed now, as of yesterday Symfony 6 stable is out https://github.com/symfony/symfony/releases/tag/v6.0.0

Reply:
I've removed that block, thanks for the reminder.